project_config.yml

```
@@ -0,0 +1,16 @@
+  learning_rate: 0.01
+  n_estimators: 1000
+  max_depth: 6
+
+num_features:
+  - trip_distance
+
+cat_features:
+  - dropoff_zip
+  - pickup_zip
+
+target: fare_amount
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)